### PR TITLE
CLDR-17560 Overall Errors

### PR DESCRIPTION
**Pull request by srl295**
CLDR-17560

- [X] This PR completes the ticket.
 
 - add new CldrError and OverallErrors components
 - add new REST endpoint for whole-locale-errors
 - remove Report for supplemental data (whole locale errors) and references thereto
 - remove link from info panel to supplemental chart
 - add whole-locale widget to General Info

ALLOW_MANY_COMMITS=true

### CODE REVIEW

**Comment by srl295** [comment]
No output if there's no issue. 

Otherwise: 

image

**Comment by srl295** [comment]
this is up on staging as of this writing

File: tools/cldr-apps/js/src/views/CldrError.vue
Comment (btangmu) [inline tools/cldr-apps/js/src/views/CldrError.vue:40]: should message, subtype and subtypeUrl be declared here? It looks like they're filled in from json from the back end. If that fails for any reason, the front end will throw cryptic errors like undefined subtype... I know we already have a lot of js code that makes this kind of implicit assumption. Still, objects with clearly defined and documented properties would be more ideal. I think status corresponds to back-end CheckCLDR.CheckStatus, and subtype corresponds to CheckCLDR.CheckStatus.Subtype. Comments could help...

File: tools/cldr-apps/js/src/views/CldrError.vue
Comment (btangmu) [inline tools/cldr-apps/js/src/views/CldrError.vue:38]: is type an object, or a string? On the back end it's an enum, but in the json it looks very much like a string such as "Warning" -- or am I just totally confused?

**Review by btangmu — APPROVED** [review]
looks good aside from some minor quibbles

File: tools/cldr-apps/js/src/esm/cldrSurvey.mjs
Comment (srl295) [reply tools/cldr-apps/js/src/esm/cldrSurvey.mjs:791]: true.

File: tools/cldr-apps/js/src/views/CldrError.vue
Comment (srl295) [reply tools/cldr-apps/js/src/views/CldrError.vue:32]: this was originally a lowercasing function. There's no titlecase in JS.

File: tools/cldr-apps/js/src/views/CldrError.vue
Comment (srl295) [reply tools/cldr-apps/js/src/views/CldrError.vue:40]: I don't think it will type check here.

File: tools/cldr-apps/js/src/views/CldrError.vue
Comment (srl295) [reply tools/cldr-apps/js/src/views/CldrError.vue:38]: It's an object.

**Comment by srl295** [comment]
do we want to merge changes to https://unicode-org.atlassian.net/browse/CLDR-17664 before merging this?

File: tools/cldr-apps/js/package.json
Comment (srl295) [reply tools/cldr-apps/js/package.json:49]: it was to fix an issue with Chrome

**Comment by srl295** [comment]
@btangmu i'm going to make this one draft because it's waiting on some further UI review

File: tools/cldr-apps/js/src/views/CldrError.vue
Comment (btangmu) [reply tools/cldr-apps/js/src/views/CldrError.vue:38]: I don't think we're talking about the same thing... `status` itself is an object, yes (I wrongly guessed CheckCLDR.CheckStatus but you say CheckStatusSummary and I believe you).

I was referring to `status.type`. On the back end that's `CheckCLDR.CheckStatus.Type`, which is an enum, and on the front end it's not an Object, it's a String. So, I think `type: Object` is wrong, it should be `type: String`

**Review by macchiati — APPROVED** [review]

**Pull request merged**
